phantomgames/Home/HomeScreenViewController.swift

```diff
@@ -33,51 +43,77 @@ class HomeScreenViewController: UIViewController{
         homeCollectionView.dataSource = self
         homeCollectionView.register(UINib(nibName: "CustomCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "CustomCollectionViewCell")
     }
+    private func setupGestureRecognizers() {
+        let pcTapGesture = UITapGestureRecognizer(target: self, action: #selector(pcImageTapped))
+        pcImage.isUserInteractionEnabled = true
+        pcImage.addGestureRecognizer(pcTapGesture)
+        
+        let browserTapGesture = UITapGestureRecognizer(target: self, action: #selector(browserImageTapped))
+        browserImage.isUserInteractionEnabled = true
+        browserImage.addGestureRecognizer(browserTapGesture)
+    }
+    @objc private func pcImageTapped() {
+        navigateToSearchGameScreen(with: Constants.Endpoints.pcGamesURL)
+    }
+    
+    @objc private func browserImageTapped() {
+        navigateToSearchGameScreen(with: Constants.Endpoints.browserGamesURL)
+    }
+    
+    private func navigateToSearchGameScreen(with url: String) {
+        let storyboard = UIStoryboard(name: "SearchGame", bundle: nil)
+        if let searchGameVC = storyboard.instantiateViewController(withIdentifier: "SearchGameViewController") as? SearchGameViewController {
+            searchGameVC.gamesURL = url
+            navigationController?.pushViewController(searchGameVC, animated: true)
+        }
+    }
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == Constants.SegueIdentifiers.GameDetailScreenSegue,
+           let destinationVC = segue.destination as? GameDetailViewController,
```

Comment:
You don't need this identifier check since this screen only has one segue
```suggestion
        if let destinationVC = segue.destination as? GameDetailViewController,
```